e2e/ansible/cleanup-openebs.yml

```diff
@@ -2,4 +2,5 @@
 - hosts: kubernetes-kubemasters
   roles:
     - {role: k8s-openebs-uninstall-helm, when: deployment_mode == "hyperconverged" and openebs_deploy == "helm"}
+    - {role: k8s-openebs-cleanup}
```

Comment:
Does this need a conditional invocation when openebs_deploy=kubectl ?

Reply:
You are right, we will need it.